services/src/bin/main.rs

```diff
@@ -97,12 +97,13 @@ fn colored_custom_log_format(
     record: &Record,
 ) -> Result<(), std::io::Error> {
     let level = record.level();
+    let style = style(level);
     write!(
         w,
         "[{}] {} [{}] {}",
-        style(level, now.now().format("%Y-%m-%d %H:%M:%S %:z")),
-        style(level, level),
+        style.paint(now.now().format("%Y-%m-%d %H:%M:%S %:z").to_string()),
+        style.paint(level.to_string()),
         record.module_path().unwrap_or("<unnamed>"),
-        style(level, &record.args())
+        style.paint(&record.args().to_string())
```

Comment:
Why can't we put references into the paint function? Why convert it with to_string?

Reply:
the argument has to be `Into<Cow>`. Calling `to_string' is how the library itself does it https://github.com/emabee/flexi_logger/blob/4dafd4e07e9b63793c4d034348241b7ea956bfb1/src/formats.rs#L49